packages/graphics/graphics.5.1.0/opam

```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
+synopsis: "The OCaml graphics library"
+description: """
+The graphics library provides a set of portable drawing
+primitives. Drawing takes place in a separate window that is created
+when Graphics.open_graph is called.
+
+This library used to be distributed with OCaml up to OCaml 4.08.
+"""
+maintainer: ["jeremie@dimino.org" "david.allsopp@metastack.com"]
+authors: [
+  "Xavier Leroy" "Jun Furuse" "J-M Geffroy" "Jacob Navia" "Pierre Weis"
+]
+license: "LGPL-2.1 with OCaml linking exception"
+homepage: "https://github.com/ocaml/graphics"
+doc: "https://ocaml.github.io/graphics/"
+bug-reports: "https://github.com/ocaml/graphics/issues"
+depends: [
+  "dune" {>= "1.11"}
```

Comment:
```suggestion
  "dune" {>= "1.11"}
  "dune-configurator"
```
Seems like dune-configurator is missing
```

#=== ERROR while compiling graphics.5.1.0 =====================================#
# context              2.0.5 | linux/x86_64 | ocaml-base-compiler.4.09.0 | pinned(https://github.com/ocaml/graphics/releases/download/5.1.0/graphics-5.1.0.tbz)
# path                 ~/.opam/4.09/.opam-switch/build/graphics.5.1.0
# command              ~/.opam/4.09/bin/dune build -p graphics -j 2 @install
# exit-code            1
# env-file             ~/.opam/log/graphics-23-1c4147.env
# output-file          ~/.opam/log/graphics-23-1c4147.out
### output ###
# File "src/discover/dune", line 3, characters 17-34:
# 3 |  (libraries unix dune-configurator))
#                      ^^^^^^^^^^^^^^^^^
# Error: Library "dune-configurator" not found.
# Hint: try: dune external-lib-deps --missing -p graphics -j 2 @install

- File "src/discover/dune", line 3, characters 17-34:
- 3 |  (libraries unix dune-configurator))
-                      ^^^^^^^^^^^^^^^^^
- Error: Library "dune-configurator" not found.
- Hint: try: dune external-lib-deps --missing -p graphics -j 2 @install
```

Reply:
Ah yes, I'll just bump the minimal version of dune to 2.0.0.

Reply:
Done